Cargo.toml

```diff
@@ -40,7 +40,7 @@ rust-version = "1.85"
 
 [workspace.dependencies]
 anyhow = "1.0.72"
-apache-avro = "0.17"
+apache-avro = "0.20.0"
```

Comment:
I'm wondering if it's better to use `0.20` (without patch version)?
Since that way, libraries which depend on iceberg-rust are legal to resolve to one avro version which is not `0.20.0`, but sth like `0.20.1`?

Reply:
Ah, I wasn't aware of that. Thanks for the suggestion 👍